ci-operator/config/openshift/cluster-kube-apiserver-operator/openshift-cluster-kube-apiserver-operator-master.yaml

```diff
@@ -32,15 +32,19 @@ test_binary_build_commands: make build GOFLAGS='-race'
 tests:
 - as: verify
   commands: |
-    go get -u golang.org/x/tools/imports
     make verify -k
   container:
     from: bin
+- as: verify-deps
+  commands: |
+    HOME=$( mktemp -d )
```

Comment:
why?

Reply:
something needed it set (I think it was glide, for the cache) but the base images don't set it

Reply:
Ya, to be clear, while verifying deps, we will pull the vendored packages the to `$HOME` and do a comparison. So, we need to set it.